.travis.yml

```diff
@@ -73,6 +74,12 @@ matrix:
       addons:
         hosts:
           - mysql2gem.example.com
+    - rvm: 2.4
```

Comment:
This part is a little bit tricky. Actually the tested Ruby on CentOS is not Ruby 2.4 but default installed Ruby on CentOS. But I was not sure how to update `rvm: foo` to show `rvm` is not used directly on Travis.